api/graphql/application_rounds/application_round_types.py

```diff
@@ -0,0 +1,207 @@
+    ApplicationRoundStatus,
+    ApplicationStatus,
+)
+from permissions.api_permissions.graphene_permissions import ApplicationRoundPermission
+from permissions.helpers import can_manage_service_sectors_application_rounds
+from spaces.models import ServiceSector
+
+from ..reservations.reservation_types import ReservationPurposeType
+
+
+class AggregatedDataType(graphene.ObjectType):
+    allocation_result_events_count = graphene.Int()
+    allocation_duration_total = graphene.Int()
+    total_reservation_duration = graphene.Int()
+    total_hour_capacity = graphene.Int()
+
+
+class ServiceSectorType(AuthNode, PrimaryKeyObjectType):
+    permission_classes = (
+        (ApplicationRoundPermission,)
+        if not settings.TMP_PERMISSIONS_DISABLED
+        else (AllowAny,)
+    )
+
+    class Meta:
+        model = ServiceSector
+        fields = ("id",)
+        interfaces = (graphene.relay.Node,)
+
+
+class ApplicationRoundBasketType(AuthNode, PrimaryKeyObjectType):
+    permission_classes = (
+        (ApplicationRoundPermission,)
+        if not settings.TMP_PERMISSIONS_DISABLED
+        else (AllowAny,)
+    )
+
+    purpose_ids = graphene.List(graphene.Int)
+    age_group_ids = graphene.List(graphene.Int)
+    home_city_id = graphene.Int()
+
+    class Meta:
+        model = ApplicationRoundBasket
+        fields = (
+            "id",
+            "name",
+            "purpose_ids",
+            "must_be_main_purpose_of_applicant",
+            "customer_type",
+            "age_group_ids",
+            "home_city_id",
+            "allocation_percentage",
+            "order_number",
+        )
+        interfaces = (graphene.relay.Node,)
+
+    def resolve_purpose_ids(self, info: graphene.ResolveInfo):
+        return self.purposes.all().values_list("id", flat=True)
+
+    def resolve_age_group_ids(self, info: graphene.ResolveInfo):
+        return self.age_groups.all().values_list("id", flat=True)
+
+    def resolve_home_city_id(self, info: graphene.ResolveInfo):
+        return getattr(self.home_city, "id", None)
+
+
+class ApplicationRoundType(AuthNode, PrimaryKeyObjectType):
+    permission_classes = (
+        (ApplicationRoundPermission,)
+        if not settings.TMP_PERMISSIONS_DISABLED
+        else (AllowAny,)
+    )
+
+    application_round_baskets = graphene.List(ApplicationRoundBasketType)
+    reservation_units = graphene.List(ReservationUnitType)
+    purposes = graphene.List(ReservationPurposeType)
+    service_sector = graphene.Field(ServiceSectorType)
+    status = graphene.Field(
+        graphene.Enum("status", ApplicationRoundStatus.STATUS_CHOICES)
+    )
+    status_timestamp = graphene.DateTime()
+
+    is_admin = graphene.Boolean()
+
+    aggregated_data = graphene.Field(AggregatedDataType)
+
+    approved_by = graphene.String()
+
+    applications_sent = graphene.Boolean()
+
+    class Meta:
+        model = ApplicationRound
+        fields = (
+            "id",
+            "name_fi",
```

Comment:
we have helper method for adding all translatable fields: `get_all_translatable_fields`
Example usage [here](https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/478/files#diff-fda3743b409d6f4efdcea4942c95df1c6f607363ccb91ad90ef806a1ddffc44fR251).

Reply:
Used helper instead of manually added fields.

Reply:
oh, the link does not work but here's an example:
```    
class Meta:
  model = ModelOfTheType
  fields = [
      "field_1",
      "field_2",
  ] + get_all_translatable_fields(model)